src/applications/income-and-asset-statement/config/submit.js

```diff
@@ -15,9 +17,26 @@ export function replacer(key, value) {
     }
   }
 
+  if (value === null) {
+    return undefined;
+  }
+
   return value;
 }
 
+export function removeDisallowedFields(form) {
```

Comment:
I might shift this logic into `replacer`, but at the moment am focused on unblocking form submissions